src/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -381,8 +400,9 @@ contract BLSRegistryCoordinatorWithIndices is EIP712, Initializable, IBLSRegistr
         address operator, 
         bytes32 operatorId,
         bytes calldata quorumNumbers,
-        string memory socket
-    ) internal virtual returns (RegisterResults memory) {
+        string memory socket,
+        SignatureWithSaltAndExpiry memory signatureWithSaltAndExpiry
```

Comment:
Would prefer we rename this to `operatorSignature`. Less wordy and more clear intention.